src/components/NavBar/__snapshots__/index.test.jsx.snap

```diff
@@ -60,13 +60,20 @@ exports[`navbar renders correctly 1`] = `
     >
       homePage.navbar.signIn
     </button>
-    <button
-      className="shadow-none mr-1 btn btn-outline-info"
-      disabled={false}
-      type="button"
+    <a
+      aria-current={null}
+      className="nav-link"
+      href="/signup"
+      onClick={[Function]}
```

Comment:
why do you call onClick like this [Function]?

Reply:
I wrapped the button with NavLink to take us to the SignUp page. I did not want to change it to not mess with the NavBar styling.

Reply:
I notice now that I was reviewing the index.test.jsx.snap. Don't take in consideration what I wrote